handlebars/.gitignore

```diff
@@ -1,28 +1,16 @@
-# Numerous always-ignore extensions
```

Comment:
Some of the rules can be useful for vim users. I don't see any reasons to remove it